components/threading/global_threads/thread_list/virtualized_thread_list.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {memo, useRef, useCallback, useMemo} from 'react';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import {DynamicSizeList} from 'dynamic-virtualized-list';
+
+import {$ID} from 'mattermost-redux/types/utilities';
+import {UserThread} from 'mattermost-redux/types/threads';
+
+import ThreadItem from '../thread_item';
+
+type Props = {
+    ids: Array<$ID<UserThread>>;
+    selectedThreadId?: $ID<UserThread>;
+};
+
+const ITEMS_ON_SCREEN = 8;
+const OVERSCAN_COUNT_BACKWARD = 4;
+const OVERSCAN_COUNT_FORWARD = 4;
+
+const virtListStyles = {
```

Comment:
do we need to have the style inline?

Reply:
`DynamicSizeList` accepts a `style` prop and not a class one. 
My guess would be that since there are no stylesheets included in the module, there are only [inline styles](https://github.com/mattermost/dynamic-virtualized-list/blob/8b239f63178925163245e2629a6dea339702b4f5/src/DynamicSizeList.js#L367-L374), 
we pass inline styles so we can properly override if needed(?).